includes/html/api_functions.inc.php

```diff
@@ -2526,35 +2530,34 @@ function maintenance_devicegroup(Illuminate\Http\Request $request)
         return api_error(404, "Device group $name not found");
     }
 
-    if (! $request->json('duration')) {
+    if (empty($data['duration'])) {
         return api_error(400, 'Duration not provided');
     }
 
-    $notes = $request->json('notes');
-    $title = $request->json('title') ?? $device_group->name;
-
+    empty($data['notes']) ? $notes = '' : $notes = $data['notes'];
```

Comment:
This is not the normal format and could be misread, lets do it the regular way
```suggestion
    $notes = $data['notes'] ?? '';
```